nestedtensor/csrc/nested_node.cpp

```diff
@@ -9,27 +9,38 @@ namespace nested_tensor {
 using namespace torch::jit;
 using namespace torch::autograd::utils;
 
-std::string _NestedNode___str__(const TensorNode& nested_node) {
+void pretty_print_tensor(std::stringstream& result, const std::string& python_tensor_string, const std::string& tabs) {
+  auto tensor = strdup(python_tensor_string.c_str());
+  auto tokens = strtok(tensor, "\n");
```

Comment:
When you call strtok, please include cstring and use [std::strtok](https://en.cppreference.com/w/cpp/string/byte/strtok) to get the C++ version of this funciton.

Reply:
good to know! thanks